sebs/gcp/gcp.py

```
@@ -412,24 +422,85 @@ def update_function(
+
+            db = (
+                cast(GCPSystemResources, self._system_resources)
+                .get_nosql_storage()
+                .benchmark_database(code_package.benchmark)
+            )
+            envs["NOSQL_STORAGE_DATABASE"] = db
+
+        return envs
+
+    def update_function_configuration(
+        self, function: Function, code_package: Benchmark, env_variables: dict = {}
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)